spyder/widgets/findreplace.py

```diff
@@ -397,8 +424,9 @@ def regexp_error_msg(pattern):
                 return str(e)
             return None
 
-        if multiline_replace_check and self.replace_widgets[0].isVisible() and \
-           len(to_text_string(self.editor.get_selected_text()).splitlines())>1:
+        if (multiline_replace_check and self.replace_widgets[0].isVisible()
+                and len(to_text_string(
+                    self.editor.get_selected_text()).splitlines())>1):

```

Comment:
Missing whitespace around ``>`` operator here, which is one of the explicitly required ones per PEP8.